### DISCUSSION TITLE
Monitoraggio fotovoltaico

### DISCUSSION THREAD

**Discussion by roby1968** [indomus/forum#113] [HUB: Home Assistant]
Salve
Avendo imminentemente un impiato forovoltaico ,
Vorrei con uno shelly em
Monitorare la produzione,l auticonsumo,e quella che mando  al enel tutto in home assistant.
Con grafici..
Ci soni guide
Grazie

**Comment by cacelotto** [comment]
Io ho inverter solaredge e leggo quei valori e anche altri con le APi dal sito di monitoraggio. Una volta si leggevano direttamente sull'inverter, poi hanno deciso che non va più bene. Il ritardo è di circa 15 minuti.
